kqueen/serializers.py

```diff
@@ -1,13 +1,20 @@
 from flask.json import JSONEncoder
 
 
-class CustomJSONEncoder(JSONEncoder):
+class KqueenJSONEncoder(JSONEncoder):
     def default(self, obj):
 
+        if hasattr(obj, 'get_dict'):
+            return obj.get_dict()
+        elif hasattr(obj, 'serialize'):
+            return obj.serialize()
+
         try:
             return JSONEncoder.default(self, obj)
```

Comment:
at last :)

Reply:
not customjsonencoder anymore

Reply:
Yes, it was a gift for you. I'm slowly starting to use it in our app and moving logic from models to JSON encoder.